opentelemetry-appender-tracing/examples/basic.rs

```diff
@@ -0,0 +1,25 @@
+//! run with `$ cargo run --example basic --all-features
+
+use opentelemetry_appender_tracing::{layer};
+use tracing::{info, error};
+use opentelemetry_sdk::{logs::{LoggerProvider, Config}, Resource};
+use opentelemetry_api::KeyValue;
+use tracing_subscriber::prelude::*;
+
+fn main() {
+    let exporter  = opentelemetry_stdout::LogExporter::default();
+    let provider: LoggerProvider = LoggerProvider::builder()
+    .with_config(
+        Config::default().with_resource(Resource::new(vec![KeyValue::new(
+            "service.name",
+            "log-appender-tracing-example",
+        )])),
+    )
+    .with_simple_exporter(exporter)
+    .build();
+    let layer  = layer::OpenTelemetryTracingBridge::new(&provider);
+    tracing_subscriber::registry().with(layer).init();
+
+    info!(fruit = "apple", price = 2.99, message = "hello from from price");
```

Comment:
This is just an initial version.
We need to discuss and finalize
1. Move this to a new repo under tokio/tracing org OR merge with tokio/tracing-opentelemetry
2. Let users configure where the events should be sent to - either as Logs or SpanEvents.

Reply:
My vote for 1) would be to merge into tracing-opentelemetry, and 2) support whichever is configured as there may be a case where traces _and_ logs would be requested as they could be sent to different exporters.

Reply:
There is an issue of potential duplication:

1. A user is currently using tracing for Spans and Events, and have currently configured tracing-opentelemetry, so they end up as OTel Spans and SpanEvents respectively.

2. The user now wants to use tracing for standalone Events (i.e outside of any Spans). They'll be using the Events API from tracing. And they setup the equivalent of this PR (so that Events gets sent as LogRecords). But this also causes a side-effect - the Events from 1, will now be sent both as SpanEvents and as LogRecords. This is a cost concern.

@jtescher Do you have any suggestions on fixing the above? I think an option in tracing-opentelemetry to ignore Events could work? By default, if user does 1,2, they get events as SpanEvents and LogRecords. If that is not desired, they can turn off SpanEvents from 1 or turn off LogRecords from 2.

Reply:
We should definitely make sure that tracing-opentelemetry is configurable about whether it should turn events into (a) SpanEvents, (b) LogRecords or (c) both.